src/pages/resultsView/plots/styles.scss

```diff
@@ -166,13 +166,23 @@
     }
 
   }
-    
+    strong.quickPlotsTitle {
+      margin-right: 5px;
+    }
+
+    .quickPlotsContainer {
+      display: flex;
+      margin-right: 5px;
+
+      .plots-tab-pills {
```

Comment:
i actually think we should apply this  padding to all the nav pills (used a fair amount in app)